feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/isis_change_lsp_lifetime_test.go

```diff
@@ -0,0 +1,317 @@
+
+package isis_change_lsp_lifetime_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+const (
+	plenIPv4       = 30
+	plenIPv6       = 126
+	isisInstance   = "DEFAULT"
+	dutAreaAddress = "49.0001"
+	ateAreaAddress = "49.0002"
+	dutSysID       = "1920.0000.2001"
+	lspLifetime    = 500
+	v4Route        = "203.0.113.0/30"
+	v6Route        = "2001:db8::203:0:113:0/126"
+	v4IP           = "203.0.113.1"
+	v6IP           = "2001:db8::203:0:113:1"
+)
+
+var (
+	dutPort1Attr = attrs.Attributes{
+		Desc:    "DUT to ATE port1 ",
+		IPv4:    "192.0.2.1",
+		IPv6:    "2001:db8::192:0:2:1",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	atePort1attr = attrs.Attributes{
+		Name:    "ATE to DUT port1 ",
+		IPv4:    "192.0.2.2",
+		IPv6:    "2001:db8::192:0:2:2",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	dutPort2Attr = attrs.Attributes{
+		Desc:    "DUT to ATE port2 ",
+		IPv4:    "192.0.2.5",
+		IPv6:    "2001:db8::192:0:2:5",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	atePort2attr = attrs.Attributes{
+		Name:    "ATE to DUT port2",
+		IPv4:    "192.0.2.6",
+		IPv6:    "2001:db8::192:0:2:6",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+)
+
+// configureDUT configures all the interfaces on the DUT.
+func configureDUT(t *testing.T) {
+	t.Helper()
+	dc := gnmi.OC()
+	dut := ondatra.DUT(t, "dut")
+
+	i1 := dutPort1Attr.NewOCInterface(dut.Port(t, "port1").Name(), dut)
+	t.Log("Pushing interface config on DUT port1")
+	gnmi.Replace(t, dut, dc.Interface(i1.GetName()).Config(), i1)
+
+	i2 := dutPort2Attr.NewOCInterface(dut.Port(t, "port2").Name(), dut)
+	t.Log("Pushing interface config on DUT port2")
+	gnmi.Replace(t, dut, dc.Interface(i2.GetName()).Config(), i2)
+
+	if deviations.ExplicitPortSpeed(dut) {
+		fptest.SetPortSpeed(t, dut.Port(t, "port1"))
+		fptest.SetPortSpeed(t, dut.Port(t, "port2"))
+	}
+}
+
+// configureISIS configures isis on DUT.
+func configureISIS(t *testing.T, dut *ondatra.DUTDevice, intfName string, dutAreaAddress, dutSysID string) {
+	t.Helper()
+	d := &oc.Root{}
+	configPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+
+	if !deviations.ISISprotocolEnabledNotRequired(dut) {
+		prot.Enabled = ygot.Bool(true)
+	}
+	isis := prot.GetOrCreateIsis()
+	globalIsis := isis.GetOrCreateGlobal()
```

Comment:
Please add deviation isis_single_topology_required similar to 
https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go#L507

Reply:
@arvbaska1, can you please address these requests in a different pull? We need to merge this one soon.